src/magic_folder/magic_folder.py

```diff
@@ -1568,6 +1570,7 @@ def _add_db_entry(filenode):
                         filecap,
                         last_downloaded_uri,
                         last_downloaded_timestamp,
+                        u'URI:DIR2-RO:0',
```

Comment:
This PR is terribly incomplete. It only implemented just the part that creates a tahoe-lafs snapshot representation, so I was faking all the db operations. Also, something I didn't make it very clear in the PR description is that since the db storage is not done yet, this was co-existing with the existing mechanisms of db operation. I thought addressing db operation changes would be a big change since all the tests would break, so I thought the upload/download of snapshots could be done as a separate PR.  So, these `URI:DIR2-RO` (I thought I changed it to `CHK` URLs as that's what they are) are fake things passed to the db to store against the file path. I was unclear on how to go about it when I started, but I have a slightly better picture now.. I think I will address some of these in this PR itself in subsequent commits.